chalk-solve/src/infer/unify.rs

```diff
@@ -674,6 +756,7 @@ impl<'t, I: Interner> Unifier<'t, I> {
         &mut self,
         lifetime: &Lifetime<I>,
         universe_index: UniverseIndex,
+        variance: Variance,
     ) -> Lifetime<I> {
```

Comment:
Lifetimes are also cloned in the `Invariant` case.

Reply:
Updated.